model/fv_regional_bc.F90

```diff
@@ -3975,12 +4214,12 @@ subroutine remap_dwinds_regional_bc(Atm                              &
 
      do k=1,km+1
         do i=is_v,ie_v
-           pe0(i,k) = ak0(k) + bk0(k)*0.5*(psc(i-1,j)+psc(i,j))
+           pe0(i,k) = ak0(k) + bk0(k)*psc(i,j)
         enddo
      enddo
      do k=1,npz+1
         do i=is_v,ie_v
-           pe1(i,k) = Atm%ak(k) + Atm%bk(k)*0.5*(psc(i-1,j)+psc(i,j))
+           pe1(i,k) = Atm%ak(k) + Atm%bk(k)*psc(i,j)
```

Comment:
Same as the previous comment.

Reply:
The j-1 value is outside the boundary in some places.

Reply:
You will find this same issue for j-1 a few lines up, in two places.

Reply:
This error has been corrected, along with the similar errors in three other places.